src/docker/docker-hub.ls

```
@@ -0,0 +1,38 @@
+require! {
+  'child_process'
+  'exosphere-shared' : { DockerHelper }
+  'path'
+  'require-yaml'
+}
+
+
+# Pushes images to DockerHub
+class DockerHub
+
+  (@app-config, @logger) ->
+
+
+  dockerhub-push: (done) -> # TODO: make sure this is run before bin/start-deploy
+    images = @_image-names!
+    for image in images
+      unless DockerHelper.image-exists image then return throw Error "No Docker image exists for service '#{image.name}'. Please run exo-setup."
```

We don't want to show a stacktrace to the user here, since this is not an error of our code. Please return an error via `done` here.